fhirstore/fhirstore.py

```diff
@@ -0,0 +1,42 @@
+import os
+import json
+from pymongo import MongoClient
+
+from fhirstore.schema import SchemaParser
+
+
+class FHIRStore():
+    def __init__(self, client, db_name: str):
+        self.db = client[db_name]
```

Comment:
what is the type of client?
Maybe if that's not to annoying we could use typed annotations as a general good practice